inc/class-wpseo-replace-vars.php

```diff
@@ -106,7 +106,10 @@ public static function register_replacement( $var, $replace_function, $type = 'a
 				trigger_error( __( 'A replacement variable can only contain alphanumeric characters, an underscore or a dash. Try renaming your variable.', 'wordpress-seo' ), E_USER_WARNING );
 			}
 			elseif ( strpos( $var, 'cf_' ) === 0 || strpos( $var, 'ct_' ) === 0 ) {
+				// Ignore invalid placeholder detection, there is no (s)printf usage in the following line.
```

Comment:
Actually, the ignores annotations can be removed as this particular file is excluded from that particular check in the ruleset with documentation to that effect: https://github.com/Yoast/wordpress-seo/pull/6875/commits/a4849f33533d7012acacf6545cc91a79ac776fd7